server/src/instant/util/s3.clj

```diff
@@ -199,39 +177,49 @@
      (->> chunks
           (mapcat #(delete-objects bucket-name (vec %)))))))
 
+(def signer-creds*
+  (delay
+    (let [access-key (config/s3-storage-access-key)
+          secret-key (config/s3-storage-secret-key)]
+      (if (and access-key secret-key)
+        {:access-key access-key
+         :secret-key secret-key
+         :region "us-east-1"}
```

Comment:
I am not 100% sure this is the best way. I am assuming the previous S3 client somehow figured out regions for us -- I am not sure how though. For now I could add the region as a config value too, if we like that approach